package.json

```diff
@@ -4,25 +4,24 @@
   "homepage": "./",
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.8.2",
-    "@emotion/styled": "^11.8.1",
-    "@mui/icons-material": "^5.5.1",
-    "@mui/material": "^5.5.1",
-    "@mui/system": "^5.5.1",
-    "@reduxjs/toolkit": "^1.7.2",
-    "@types/node": "^12.20.48",
-    "@types/react": "^16.14.25",
-    "@types/react-dom": "^16.9.14",
-    "@types/react-redux": "^7.1.22",
```

Comment:
react-redux was updated to v8 but this one is still on version 7. v8 has converted the project to typescript, so this may not be needed anymore.